### PR TITLE
[LIGHTS] Implement basic light

### PR DESCRIPTION
**Pull request by aleemont1**
This PR implements a basic Light device.
I'm thinking about creating an Adjustable version, too, but for now I'd stick with Toggleable.

### CODE REVIEW

**Comment by michelececcacci** [comment]
i already left some comments on #51

File: app/src/main/java/homer/model/lights/Light.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Override
+    public boolean isToggled() {
+        return this.getState();
+    }
+
+    @Override
+    public void toggle() {
+        this.state = (this.isToggled() ^ true);
```
Comment (michelececcacci) [inline app/src/main/java/homer/model/lights/Light.java:47]: could be simplified to `this.state ^= true`

Comment (aleemont1) [reply app/src/main/java/homer/model/lights/Light.java:47]: In fact it was like that at first, but I thought it was more explicit/readable this way.
Is it a violation of some principle or best-practice?

**Comment by michelececcacci** [comment]
In my opinion my suggestion makes it more explicit, that's it. But if you
like yours more, merge it that way.It's not a major issue.

**Pull request merged**